spring-cloud-openfeign-async-dependencies/pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>spring-cloud-dependencies-parent</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>3.0.3-SNAPSHOT</version>
+		<relativePath/>
+	</parent>
+	<artifactId>spring-cloud-openfeign-async-dependencies</artifactId>
+	<version>0.1.0-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<name>Spring Cloud OpenFeign Async Dependencies</name>
+	<description>Spring Cloud OpenFeign Async Dependencies</description>
+	<properties>
+		<feign.version>11.1</feign.version>
+		<feign-form.version>3.8.0</feign-form.version>
+	</properties>
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>io.github.openfeign</groupId>
+				<artifactId>feign-bom</artifactId>
```

Comment:
maybe import the s-c-openfeign bom, then you don't have to manage the feign versions.